src/graph-descriptor/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = class GraphDescriptor extends React.Component {
             node={child}
             steps={[idx === children.length - 1]}
             path={[child]}
+            isCollapsed={this.props.isCollapsed || false}
```

Comment:
If I'm reading it right, the `|| false` here will break collapse behavior for non-SSR use cases since the prop will never be undefined.